Tasks/AzureFunctionAppV1/deploymentProvider/ConsumptionWebAppDeploymentProvider.ts

```diff
@@ -30,6 +38,71 @@ export class ConsumptionWebAppDeploymentProvider extends AzureRmWebAppDeployment
         }
         return storageData;
     }
+
+    private async uploadPackage(storageDetails, deployPackage) : Promise<string> {
+        var defer = Q.defer<string>();
+        let storageAccount = storageDetails["AccountName"];
+        let storageKey = storageDetails["AccountKey"];
+        const blobService = azureStorage.createBlobService(storageAccount, storageKey);
+
+        const containerName = 'azure-pipelines-deploy';
```

Comment:
specify `type` wherever possible.
` const containerName: string = 'azure-pipelines-deploy';`

Reply:
It is done